packing-api.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+# Script to pack api app for severless or micro contanier service
+echo 'Building ... \n' && yarn workspace @ehpr/api build && \
```

Comment:
Might be a good idea to clear the previous  zip file.

Reply:
Zip is cleaned in following steps